packages/docusaurus-plugin-content-docs/src/plugin-content-docs.d.ts

```diff
@@ -356,6 +361,8 @@ declare module '@docusaurus/plugin-content-docs' {
     pagination_prev?: string | null;
     /** Should this doc be excluded from production builds? */
     draft?: boolean;
+    /** Front matter override for last update to doc */
```

Comment:
This JSDoc needs to be synchronized with the documentation.

Reply:
Sorry, I've never worked with JSDoc before, so I don't fully understand what that means. Is it something like this?
```js
/** Front matter override for last update to doc 
 * @see {@link LastUpdateData}
*/
last_update?: FileChange;
```

Reply:
Your documentation for this is:

> Allows overriding the last updated author and/or date. Date can be any [parsable date string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse).

I'm asking you to port that to here, like:

```suggestion
    /**
     * Allows overriding the last updated author and/or date. Date can be any
     * [parsable date string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse).
     */
```

Reply:
Ohhh, I really over-thought that. Thank you for the clarification!